src/app/graphql/mutations/removeStar.gql

```diff
@@ -0,0 +1,10 @@
+mutation ($starrableId: ID!) {
+  removeStar(input: {starrableId: $starrableId}) {
+    starrable {
```

Comment:
@bhoomika08 Always fetch id when you are trying to fetch any object information

Reply:
fetched id of starrable.